LEGALNOTICE.md

```diff
@@ -0,0 +1,69 @@
+COPYRIGHT
+---------
+
+OWASP Zed Attack Proxy (ZAP)
+
+The software package is:
+
+    Copyright (C) 2010  Simon Bennetts
```

Comment:
I'll make that change but IANAL, so I can't advise you of the implications of not declaring a legal entity as the copyright holder. Perhaps something to reconsider if/when the team ever decides to seek trademarks.